app/models/miq_report/seeding.rb

```diff
@@ -47,17 +47,28 @@ def seed_record(path, report)
         _log.info("#{report.new_record? ? "Creating" : "Updating"} MiqReport #{filename.inspect}")
 
         yml   = YAML.load_file(path).symbolize_keys
+        name = yml[:menu_name].strip
+
         attrs = yml.slice(*column_names_symbols)
         attrs.delete(:id)
         attrs[:filename]      = filename
         attrs[:file_mtime]    = mtime
-        attrs[:name]          = yml[:menu_name].strip
+        attrs[:name]          = name
         attrs[:priority]      = File.basename(path).split("_").first.to_i
         attrs[:rpt_group]     = File.basename(File.dirname(path)).split("_").last
         attrs[:rpt_type]      = "Default"
         attrs[:template_type] = path.start_with?(REPORT_DIR.to_s) ? "report" : "compare"
 
+        begin
         report.update_attributes!(attrs)
+        rescue ActiveRecord::RecordInvalid
+          duplicate = find_by(:name => name)
+          if duplicate&.rpt_type == "Custom"
```

Comment:
😩 Nooooo... I hate this syntax...

(pedantic/personal preference)

Reply:
Out of curiosity what is the syntax you'd prefer (FWIW I like this syntax :) )

Reply:
@Fryguy while it is "Rails specific", and I usually would reach for something out of STDLIB, `.try` would be preferred, or just explicitly doing it via a `not duplicate.nil? and duplicate.rpt_type == "Custom"`.

The `&` symbol is already overloaded in my opinion, and is used for `AND` (&&), and bit-wise `AND` (&) already, so I just think this is a obscure syntax that doesn't help with readability in any form.  Would have preferred a more verbose method like `attempt()` even over a single character operator. `</rant>` `</2cents>`

Reply:
You would think with a name like lonely operator, it would attract more friends.

I don't like how it looks like invalid syntax or how it tries to make ugliness less ugly but besides that, I can `dig` it.